<launch>
  <!-- <include file="$(find bwi_kr_execution)/launch/bwi_kr_execution.launch"></include> -->

  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"></include>
  <node name="mjpeg_server" pkg="mjpeg_server" type="mjpeg_server">
    <param name="__image_transport" value="compressed" />
  </node>

  <node name="go_to_location_service_node" pkg="bwi_virtour" type="go_to_location_service_node" />
  <node name="rotate_service_node" pkg="bwi_virtour" type="rotate_service_node" />
  <node name="broadcastTour" pkg="bwi_virtour" type="broadcastTour.sh" />
</launch>
